src/app/public/src/modules/restricted-view/auth.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { BBAuthGetTokenArgs } from '@blackbaud/auth-client';
+import { BehaviorSubject } from 'rxjs';
+import { SkyAuthTokenProvider } from '@blackbaud/skyux-builder/runtime';
```

Comment:
Imports should follow the SKY UX guidelines for imports, 
see: https://docs.blackbaud.com/skyux-team-docs/style#imports